### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by MatthewMiddlehurst**
The current README is a disclaimer and needs to be replaced.

This PR adds some buttons we previously had and links to discussion areas.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,196 @@
-## DISCLAIMER: This is a fork of the sktime repository. Work is ongoing to change links and documentation to reflect this. We do not recommend using this repository in any production setting, but welcome any contributors willing to help us update the project.
+<a href="https://scikit-time.org"><img src="https://github.com/scikit-time/scikit-time/blob/main/docs/source/images/scikit-time-logo.jpg?raw=true)" width="175" align="right" /></a>
```
Comment (GuzalBulatova) [inline README.md:1]: Screenshot 2023-02-16 at 23 18 50
Doesn't render for me, is it just on my end?

File: README.md [hunk 2]
```diff
@@ -1 +1,196 @@
...
+| **Community** | [![!slack](https://img.shields.io/static/v1?logo=slack&label=slack&message=chat&color=lightgreen)](https://join.slack.com/t/scikit-time-group/shared_invite/zt-1cghagwee-sqLJ~eHWGYgzWbqUX937ig) [![!slack](https://img.shields.io/static/v1?logo=linkedin&label=LinkedIn&message=news&color=lightblue)](https://www.linkedin.com/company/scikit-time/) [![!twitter](https://img.shields.io/static/v1?logo=twitter&label=Twitter&message=news&color=lightblue)](https://twitter.com/scikit-time_toolbox)                                                                                                                                                  |
```
Comment (GuzalBulatova) [inline README.md:18]: Slack throws a "there's been a glitch" error now if I click the link
LinkedIn page "isn't available", Twitter too

File: README.md [hunk 3]
```diff
@@ -1 +1,196 @@
...
+| :star: **[Tutorials]**                 | New to scikit-time? Here's everything you need to know!  |
+| :clipboard: **[Binder Notebooks]**     | Example notebooks to play with in your browser.          |
+| :woman_technologist: **[User Guides]** | How to use scikit-time and its features.                 |
+| :scissors: **[Extension Templates]**   | How to build your own estimator using scikit-time's API. |
+| :control_knobs: **[API Reference]**    | The detailed reference for scikit-time's API.            |
+| :hammer_and_wrench: **[Changelog]**    | Changes and version history.                             |
+| :deciduous_tree: **[Roadmap]**         | scikit-time's software and community development plan.   |
+| :pencil: **[Related Software]**        | A list of related software.                              |
+
+[tutorials]: https://www.scikit-time.org/en/latest/tutorials.html
```
Comment (GuzalBulatova) [inline README.md:33]: Is it a placeholder there for the logo on scikit-time website? 
Screenshot 2023-02-16 at 23 22 56

File: README.md [hunk 4]
```diff
@@ -1 +1,196 @@
...
+| :bug: **Bug Reports**                                      | [GitHub Issue Tracker] |
+| :sparkles: **Feature Requests & Ideas**                    | [GitHub Issue Tracker] |
+| :woman_technologist: **Usage Questions**                   | [GitHub Discussions]   |
+| :speech_balloon: **General Discussion**                    | [GitHub Discussions]   |
+| :factory: **Contribution & Development**                   | [Slack]                |
+| :globe_with_meridians: **Community collaboration session** | ?                      |
+
+[github issue tracker]: https://github.com/scikit-time/scikit-time/issues
+[github discussions]: https://github.com/scikit-time/scikit-time/discussions
+[slack]: https://join.slack.com/t/scikit-time-group/shared_invite/zt-1cghagwee-sqLJ~eHWGYgzWbqUX937ig
```
Comment (GuzalBulatova) [inline README.md:56]: This link is faulty on my end as well

File: README.md [hunk 5]
```diff
@@ -1 +1,196 @@
...
+Questions and feedback are extremely welcome! Please understand that we won't be able to provide individual support via email. We also believe that help is much more valuable if it's shared publicly, so that more people can benefit from it.
+
+| Type                                                       | Platforms              |
+|------------------------------------------------------------|------------------------|
+| :bug: **Bug Reports**                                      | [GitHub Issue Tracker] |
+| :sparkles: **Feature Requests & Ideas**                    | [GitHub Issue Tracker] |
+| :woman_technologist: **Usage Questions**                   | [GitHub Discussions]   |
+| :speech_balloon: **General Discussion**                    | [GitHub Discussions]   |
+| :factory: **Contribution & Development**                   | [Slack]                |
+| :globe_with_meridians: **Community collaboration session** | ?                      |
```
Comment (GuzalBulatova) [inline README.md:52]: Maybe put Slack here for now?

**Review by GuzalBulatova — CHANGES_REQUESTED** [review]
Thank you very much, @MatthewMiddlehurst , great work! I think this is ready as soon as the faulty links are replaced.

**Review by TonyBagnall — COMMENTED** [review]
I suggest putting a link to the new slack in the disclaimer and directing people there (in addition to the permanent slack link)

File: README.md (hunk 3 above)
Comment (MatthewMiddlehurst) [reply README.md:33]: yes replaced all the sktime logos in a previous commit

File: README.md (hunk 5 above)
Comment (MatthewMiddlehurst) [reply README.md:52]: Think we can just remove it for now, can add it again when we are ready to host such sessions

**Review by aiwalter — APPROVED** [review]
Probably not the final state of README but an improvement - so approving

**Review by patrickzib — APPROVED** [review]
Thank you!

**Review by GuzalBulatova — APPROVED** [review]
Looks ready now, thank you @MatthewMiddlehurst

**Pull request merged**